service/DevHome.Service/DevHomeService.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.InteropServices;
+
+namespace DevHome.Service.Runtime;
+
+[ComVisible(true)]
+public class DevHomeService : IDevHomeService
+{
+    public DevHomeService()
+    {
+        ComHelpers.VerifyCaller();
+
+        // Track our caller process
+        ServiceLifetimeController.RegisterProcess(ComHelpers.GetClientProcess());
+    }
+
+    public int GetNumber()
+    {
+        return 42;
```

Comment:
Can you add more details as to what this method is for and why 42?

Reply:
Because it's the answer to the meaning of life. :)

It's just here so we can do a review of the infrastructure of the NT service itself and confirm data can flow back and forth. The next PR I send will remove this method and have a functional use of the service.